Objects/unicodeobject.c

```diff
@@ -2332,27 +2332,25 @@ PyUnicodeWriter_WriteUCS4(PyUnicodeWriter *pub_writer,
 }
 
 
-static int
-unicode_export(PyObject *unicode, Py_buffer *view, uint32_t *pformat,
+static int32_t
+unicode_export(PyObject *obj, Py_buffer *view,
                Py_ssize_t len, const void *buf,
-               int itemsize, const char *format, uint32_t internal_format)
+               int itemsize, const char *format, int32_t internal_format)
 {
-    if (PyBuffer_FillInfo(view, unicode, (void*)buf, len,
+    if (PyBuffer_FillInfo(view, obj, (void*)buf, len,
                           1, PyBUF_SIMPLE) < 0) {
-        *pformat = 0;
         return -1;
     }
     view->itemsize = itemsize;
     view->format = (char*)format;
     view->internal = (void*)(uintptr_t)internal_format;
```

Comment:
No need to set `view->internal`. No need to pass `internal_format` through `unicode_export()`.